digdag-docs/src/operators/require.md

```diff
@@ -74,7 +74,7 @@
 
   *rerun_on* control require> really kicks or not if the attempt for the dependent workflow already exists. 
   * *none* ... Not kick the workflow if the attempt already exists.
-  * *failed* ... Kick the workflow if the attempt exists and its result is not success.
+  * *failed* ... Kick the workflow if the attempt may exists and its result is not success.
```

Comment:
Sorry, I can't understand the meaning of using _'may'_. in this description.

BTW, your original concern is `rerun_on: failed` with the situation when the attempt does not exist.
But rerun_on is only effective if the attempt exists as described. 
If it is confused or uncertain, adding description to rerun_on may be better as follows:

```
*rerun_on* control require> really kicks or not if the attempt for the dependent workflow already exists. 
If the attempt does not exist, the dependent workflow is always kicked.
```

Reply:
> If the attempt does not exist, the dependent workflow is always kicked.

sounds great!